resource-managers/yarn/pom.xml

```diff
@@ -29,7 +29,7 @@
   <name>Spark Project YARN</name>
   <properties>
     <sbt.project.name>yarn</sbt.project.name>
-    <jersey-1.version>1.9</jersey-1.version>
+    <jersey-1.version>1.19</jersey-1.version>
```

Comment:
Upgrade jersey to 1.19. otherwise:
```
[info]   Cause: java.lang.NoClassDefFoundError: com/sun/jersey/spi/container/servlet/ServletContainer
[info]   at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.startWepApp(ResourceManager.java:1103)
[info]   at org.apache.hadoop.yarn.server.resourcemanager.ResourceManager.serviceStart(ResourceManager.java:1271)
[info]   at org.apache.hadoop.service.AbstractService.start(AbstractService.java:194)
```